README.md

```diff
@@ -87,91 +108,80 @@ echo -e "admin\n${CONCOURSE_ATC_PASSWORD}" | \
    ${FLY_CMD} -t ${DEPLOY_ENV} login -k -c "https://${DEPLOY_ENV}-concourse.cf.paas.alphagov.co.uk"
 ```
 
-## BOSH and Cloud Foundry
+### Destroy
 
-### Microbosh deployment from concourse bootstrap
+Run the `destroy-deployer` pipeline from your *Bootstrap Concourse*.
 
-These pipelines will deploy/destroy a microbosh using bosh-init. It will use the IAM profile `deployer-concourse` that was given to the Concourse instance.
```

Comment:
We depend a lot on the IAM profiles but we don't describe their names nor the permissions. This should be documented somewhere as without it this project is unusable.

I think the minimum would be a list of the profile names and just a summary of the permissions, like "read-write access to S3". Ideally, we could link to a separate doc that has all the details.